### PR TITLE
Enhancement/aws rewrite image actions

### PR DESCRIPTION
**Pull request by Tirla-Alin**
Refactored all image operations to use query api.

### CODE REVIEW

**Comment by Tirla-Alin** [comment]
@Bonczidai review this when you have time.

File: cs-jclouds/src/main/java/io/cloudslang/content/jclouds/actions/images/DeregisterImageAction.java [hunk 1]
```diff
@@ -53,34 +52,37 @@
...
-        CommonInputs inputs = new CommonInputs.CommonInputsBuilder()
-                .withProvider(provider)
-                .withEndpoint(endpoint)
-                .withIdentity(identity)
-                .withCredential(credential)
-                .withProxyHost(proxyHost)
-                .withProxyPort(proxyPort)
-                .withDebugMode(debugMode)
-                .build();
+            CustomInputs customInputs = new CustomInputs.CustomInputsBuilder()
```
Comment (lucian-cm) [inline cs-jclouds/src/main/java/io/cloudslang/content/jclouds/actions/images/DeregisterImageAction.java]: `Builder` the name of the builder is without any other prefix

File: cs-jclouds/src/main/java/io/cloudslang/content/jclouds/factory/helpers/ImageHelper.java [hunk 2]
```diff
@@ -0,0 +1,183 @@
...
+import static io.cloudslang.content.jclouds.entities.constants.Constants.AwsParams.OPERATION_TYPE;
+import static io.cloudslang.content.jclouds.entities.constants.Constants.AwsParams.USER_GROUP;
+import static io.cloudslang.content.jclouds.entities.constants.Constants.AwsParams.USER_ID;
+
+
+/**
+ * Created by Tirla Alin.
+ * Date: 30/9/2016.
+ */
+public class ImageHelper {
```
Comment (lucian-cm) [inline cs-jclouds/src/main/java/io/cloudslang/content/jclouds/factory/helpers/ImageHelper.java]: `ImageUtils` is a little better than `ImageHelper`

Comment (Tirla-Alin) [inline cs-jclouds/src/main/java/io/cloudslang/content/jclouds/factory/helpers/ImageHelper.java]: Currently all other structures (network, instances etc) have "helpers". Should i rename it?

Comment (lucian-cm) [inline cs-jclouds/src/main/java/io/cloudslang/content/jclouds/factory/helpers/ImageHelper.java]: Yes

**Comment by lucian-cm** [comment]
Changes were done. Merging

**Pull request merged**